nginx.conf

```diff
@@ -286,4 +286,8 @@ server {
   rewrite ^/platform/actors/development/continuous-integration$  /platform/actors/development/deployment/continuous-integration redirect;
   rewrite ^/platform/actors/development/testing-and-maintenance$ /platform/actors/development/deployment/automated-tests redirect;
   rewrite ^/platform/actors/development/source-code$             /platform/actors/development/deployment/source-types redirect;
+
+  # Removed pages
```

Comment:
Removing GPT plugins as they are about to stop working today - @jancurn do we have a better redirect?

Reply:
IMO this is good, cheers